source/leuven/superdome_quick_start.rst

```diff
@@ -1,77 +1,44 @@
 Superdome quick start guide
 ===========================
 
-The :ref:`Superdome <Superdome hardware>` is a shared memory machine, i.e.,
-it can be used to run multi-threaded application that require large amounts
-of RAM.
+The :ref:`Superdome <Superdome hardware>` is a shared memory machine
+which is intended for applications requiring large amounts of RAM
+on a single host.
 
-
-How to connect to Superdome?
-----------------------------
-
-Superdome does not have a dedicated login node, so in order to work with the
-system users can connect to the same login node used for Genius.
-
-.. include:: tier2_hardware/genius_login_nodes.rst
+As Superdome is part of the Genius cluster, please also see the
+:ref:`Tier-2 access page<tier2_login_nodes>` for general information
+on how to access Genius.
 
 How to run a job on Superdome?
 ------------------------------
 
-Jobs can be submitted from the login node, however, there are a few crucial
-differences.  To submit jobs to Superdome you need to
+To submit jobs to Superdome from a Genius login node you need to select either
+the ``superdome`` or ``superdome_long`` Slurm partition, depending on whether
+your job needs less than 3 days or less than 7 days.
 
-- specify the partition ``superdome``,
-- specify the queue ``qsuperdome``, and
-- use ``-L`` for specifying resources.
+The CPU resources on Superdome are furthermore allocated on the basis of entire
+sockets and not on the basis of individual cores as on other partitions.
+Since the 112 cores in Superdome are grouped in 8 sockets (see the
+:ref:`Superdome hardware page<Superdome hardware>`), this means that all
+Superdome jobs get access to a number of cores which is a multiple of 14.
 
-The resource specification is specified in terms of ``tasks``, ``lprocs``
-and ``place``.
+The allocation for the following job will hence consist of an entire socket
+(and its 14 cores)::
 
-``tasks``
-   For Superdome, the number of tasks is always equal to 1.
-``lprocs``
-   The number of logical processors is essentially the number of cores
-   you want to use for your job.
-``place``
-   The place determines where the logical processors are executed, and
-   is ``numanode`` for Superdome.  Each ``numanode`` has 14 cores and 750 GB of
-   RAM, so if you want to use, e.g., 28 cores, you would specify
-   ``lprocs=28:place=numanode=2``.
+  $ sbatch -M genius -p superdome -A myaccount --ntasks=1 myscript.slurm
 
+The following job will get two sockets (and so 28 cores in total)::
```

Comment:
By default, each task will be launched on a separate socket. The following job will therefore get two...

Reply:
Yes, would be good to mention that already there (done in 592b31402e100be571bcb942ed54d1424c33d1be)